jenkins/runRTAtest.fish

```diff
@@ -2,6 +2,9 @@
 set -l fish_trace on
 source jenkins/helper/jenkins.fish
 
+# https://stackoverflow.com/questions/56104472/why-would-setting-export-openblas-num-threads-1-impair-the-performance
+set -xg OPENBLAS_NUM_THREADS 1
+
```

Comment:
I think we should only set this if `ASAN=true` so please move this to the if in line 17-20

Reply:
I am not sure if there is a difference here between ASAN and TSAN?